src/app/modules/ix-forms/components/ix-dynamic-form/ix-dynamic-form-item/ix-dynamic-form-item.component.html

```diff
@@ -36,53 +32,48 @@
     </ix-list-item>
   </ix-list>
   <ix-input
-    *ngIf="dynamicSchema.type === DynamicFormSchemaType.Input"
+    *ngIf="dynamicSchema.type === DynamicFormSchemaType.Input && !isHidden"
     [formControlName]="dynamicSchema.controlName"
     [label]="dynamicSchema.title | translate"
     [placeholder]="dynamicSchema.placeholder | translate"
     [tooltip]="dynamicSchema.tooltip | translate"
     [required]="dynamicSchema.required"
-    [fxHide]="isHidden || dynamicSchema.hidden"
-    [class.disabled]="dynamicSchema.editable === false"
+    [disabled]="dynamicSchema.editable === false"
```

Comment:
This now adds a console warning:
```
 It looks like you're using the disabled attribute with a reactive form directive. If you set disabled to true
  when you set up this control in your component class, the disabled attribute will actually be set in the DOM for
  you. We recommend using this approach to avoid 'changed after checked' errors.
```